### PR TITLE
Fix a few assertions are not executed properly

### PR DESCRIPTION
**Pull request by nkzawa**

### CODE REVIEW

File: test/socket.io.js [hunk 1]
```diff
@@ -636,11 +636,10 @@ describe('socket.io', function(){
       });
       function getClients() {
         sio.of('/chat').clients(function(error, sids) {
-          expect(error).to.be.undefined;
```
Comment (rase-) [inline test/socket.io.js]: I'm wondering if it's cleaner to just do

```js
expect(error).to.be(undefined)
```

instead.

Comment (nkzawa) [inline test/socket.io.js]: It doesn't matter either way for me. But the value of `error` is actually `null`, so `.to.not.be.ok()` would be better.

Comment (rase-) [inline test/socket.io.js]: Ah. Aye.

Comment (nkzawa) [inline test/socket.io.js]: fixed and rebased.

**Comment by nkzawa** [comment]
umm, why the ci build failed?

**Comment by nkzawa** [comment]
I restarted the build and it's fine now.

**Pull request merged**